website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -28,13 +28,14 @@ The dbt Cloud CLI is available in all [deployment regions](/docs/cloud/about-clo
 - Ensure you are using dbt version 1.5 or higher. Refer to [dbt Cloud versions](/docs/dbt-versions/upgrade-core-in-cloud) to upgrade.
 - Avoid using SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections.
 - Avoid using [PrivateLink](/docs/cloud/secure/about-privatelink).
+- Avoid using Single sign-on (SSO) for Snowflake or SSH tunnels for Redshift.
```

Comment:
hey @aiguofer ! i've added this bit here per the slack discussion but wanted to clarify if you meant avoid using sso for snowflake only? or also all other adapters? so:

- Ensure you are using dbt version 1.5 or higher. Refer to [dbt Cloud versions](/docs/dbt-versions/upgrade-core-in-cloud) to upgrade.
- Avoid using SSH tunneling for [Postgres and Redshift](/docs/cloud/connect-data-platform/connect-redshift-postgresql-alloydb) connections.
- Avoid using [PrivateLink](/docs/cloud/secure/about-privatelink).
- Avoid using Single sign-on (SSO) for Snowflake - WHAT ABOUT OTHER ADAPTERS?

Reply:
I believe this limitation would apply to all SSO integrations.